packages/frontend/src/components/Explorer/ExploreTree/TableTree/Tree/TreeSingleNodeActions.tsx

```diff
@@ -111,16 +111,33 @@ const TreeSingleNodeActions: FC<Props> = ({
     const duplicateCustomMetric = (customMetric: AdditionalMetric) => {
         const newDeepCopyItem = JSON.parse(JSON.stringify(customMetric));
         let newId = uuidv4();
+        let newIdSubstring=newId.replace(/-/g, '').substring(0, 16);
+        let currentName=newDeepCopyItem.name;
+        const tildeIndex = currentName.indexOf('~$');
```

Comment:
I don't like much this `~$` separator... 
This will not be compatible with our `friendlyName` function to normalize ids that takes aprox [a-zA-Z0-9_]

I can see this becoming an issue on some features like reference lines or CSV exports.

Reply:
Hey @rephus,

I'm currently using "~$" as a pattern to replace the IDs (i.e ~$ after this pattern), assuming it's unique since no one would likely use this in a title. I'm considering switching to something like "_9id8_" instead. Does this sound okay to you?

Reply:
yeah, I think that is a better replacement.